src/components/implementation/tests/unit_defcompinfo/Makefile

```diff
@@ -0,0 +1,10 @@
+C_OBJS=unit_defcompinfo.o
+ASM_OBJS=cos_asm_scheduler.o
+COMPONENT=unit_defci.o
+INTERFACES=
+DEPENDENCIES=
+IF_LIB=
+ADDITIONAL_LIBS=-lcobj_format -lcos_defcompinfo_api -lcos_kernel_api
```

Comment:
There is non-consistent naming here.  `cos_kernel_api` and `cos_defcompinfo_api` don't feel like two versions of the same thing...

Perhaps `cos_defkernel_api`?  We need to change the file names accordingly.